docs/blogs/IntroGatewayAPI/IntroGatewayAPI.md

```diff
@@ -0,0 +1,165 @@
+offers in the well-established [OpenShift Service Mesh (OSSM)](https://docs.openshift.com/container-platform/4.11/service_mesh/v2x/ossm-about.html#ossm-servicemesh-overview_ossm-about)
+product line.  Having a tie to OSSM ensures a healthy outlook,  consistent experience for users, and smoother integration
+for customers who also want to use service mesh as a part of their service-networking configuration.
+
+## Why use Gateway API?
+
+GWAPI is focused on creating a well-designed, community-supported, and portable API.  It has been reviewed and implemented
+by [a large community](https://gateway-api.sigs.k8s.io/implementations/), including several vendors, proving that the API
+portability has significant appeal, enhances third-party integration, and reduces vendor lock-in.  GWAPI design is
+expressive, incorporating a native feature framework that replaces the use of custom annotations in Ingress.  At the same
+time, the design supports extensibility and the future of features through granular customization levels.  To simplify
+operations, GWAPI design focuses on role-oriented API resources, allowing for role-based access to just-enough Kubernetes
+details to satisfy and empower each organizational role.  Besides simplifying operations, role-oriented resource access
+reduces costs by enabling the safe sharing of infrastructure without requiring shared control.
+
+![Gateway API Role-Oriented Model](images/GatewayRoles.png)
+
+The key reason to use Gateway API with OpenShift Networking is that it enables the installation and use of multiple
+third-party GWAPI implementations on the same cluster.  This allows infrastructure providers to offer a range of gateway
+implementations for the benefit of cluster operators and application developers.  Also, as the standard implementations
+converge, the same GWAPI compliant configuration can be used for two different implementations on the same cluster.
+
+![Cluster with multiple Gateway API implementations](images/MultiGWAPI.png)
+
+### Features offered in Dev Preview
+
+In the Developer Preview we have chosen to provide the current version of GWAPI, 0.5.1, and the Istio release (1.15.1)
+that incorporates it. This release includes the following features:
+
+Feature | Definition
+-------------------|---------------------------------------------------------------------------------------------
+Path-based routing | Routes with different paths can have different destinations.|
+Weighted backends for traffic splitting | [Traffic Splitting]( https://gateway-api.sigs.k8s.io/guides/traffic-splitting/?h=weight#http-traffic-splitting) via weight configuration placed on backends, e.g. split 50% on each of two backends.
+Route sharding for isolation and performance | Use [GatewayAddress](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.GatewayAddress) to bind Gateways to a specific shard.
```

Comment:
Not sure whether this is copypasta, but I don't understand the connection between sharding and `GatewayAddress`.

Reply:
I found something from the documentation and I can clean it up: "The Addresses field (an array of `GatewayAddress`) represents a request for the address(es) on the “outside of the Gateway”, that traffic bound for this Gateway will use. This could be the IP address or hostname of an external load balancer or other networking infrastructure, or some other address that traffic will be sent to....The implementation MUST bind all Listeners to every GatewayAddress that it assigns to the Gateway and add a corresponding entry in GatewayStatus.Addresses."

Reply:
I guess that makes sense.  I'm looking at it from the perspective of how someone would configure sharding in terms of gateways and routes, and this text is talking about how someone would configure sharding in terms of gateways and external infrastructure.  Both parts are needed: The gateway typically needs a distinct address for the shard, and then routes need to bind to that gateway to be published as part of that shard.

Reply:
> enables you to select which envoy proxy a Gateway object uses,
Envoy and automated deployment is implementation-specific, so I'm not going to mention either of those here.  Also, this is not an instruction section, it just briefly lists dev preview features.  

I'm going to remove it as a "feature" because they aren't very comparable.